api/src/controllers/PersonaIdentifierController.js

```diff
@@ -93,15 +101,20 @@ const upsertPersonaIdentifier = catchErrors(async (req, res) => {
     authInfo
   });
 
+  const organisation = getOrgFromAuthInfo(authInfo);
   const toPersona = req.body.persona;
   if (!toPersona) {
     try {
       // if we had no persona, attempt to create an ident with a new persona
       const { identifier } = await req.personaService.createUpdateIdentifierPersona({
         ifi: req.body.ifi,
-        organisation: getOrgFromAuthInfo(authInfo),
-        personaName: req.body.ifi,
+        organisation,
+        personaName: JSON.stringify(req.body.ifi, null, 2),
```

Comment:
We could use an existing method for this - remind me to dig it up

Reply:
Reminding you to dig this up?